packages/now-build-utils/src/detectors/jekyll.ts

```diff
@@ -0,0 +1,15 @@
+import { DetectorParameters, DetectorResult } from '../types';
+
+export default async function detectJekyll({
+  fs: { exists },
+}: DetectorParameters): Promise<DetectorResult> {
+  const hasConfig = await exists('_config.yml');
+  if (!hasConfig) {
+    return false;
+  }
+  return {
+    buildCommand: ['jekyll', 'build'],
+    buildDirectory: '_site',
```

Comment:
You need to read the `_config.yml` file to find the output directory and assign it like

```js
buildDirectory: yamlConfig['destination']
```

https://jekyllrb.com/docs/configuration/options/